internal/unionstore/memdb_arena.go

```diff
@@ -320,7 +321,12 @@ func (l *memdbVlog) appendValue(nodeAddr memdbArenaAddr, oldValue memdbArenaAddr
 	return addr
 }
 
-func (l *memdbVlog) getValue(addr memdbArenaAddr) []byte {
+func (addr memdbArenaAddr) AfterCheckPoint(cp MemDBCheckpoint) bool {
+	return addr.idx > uint32(cp.blocks)-1 || (addr.idx == uint32(cp.blocks)-1 && addr.off > uint32(cp.offsetInBlock))
```

Comment:
Would it be possible the `cp.blocks` is zero and this function is called?

Reply:
Ah maybe. Actually there is a similar function `canModify` and I've removed `AfterCheckPoint`.

Reply:
Should we remove this new function too?

Reply:
Sure